### PR TITLE
Add kitchen setup recipes from Utils

### PR DESCRIPTION
**Pull request by fayvor**
Copied from the `utils` repo:
- Add Componenents recipes
- Add Getting Started recipes
- Overwrite README
- Add LICENSE
- Add setup.py

### CODE REVIEW

File: recipes/Getting_Started/Getting_Started_with_WatsonX.ipynb [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+   "metadata": {},
+   "source": [
+    "### Choose a model\n",
+    "\n",
+    "The Granite models available on WatsonX are listed [here](https://www.ibm.com/products/watsonx-ai/foundation-models)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
```
Comment (rawkintrevo) [inline recipes/Getting_Started/Getting_Started_with_WatsonX.ipynb]: same comment with execution counts

Comment (fayvor) [reply recipes/Getting_Started/Getting_Started_with_WatsonX.ipynb]: Found a command that will clear outputs:
`jupyter nbconvert --ClearOutputPreprocessor.enabled=True --inplace --to=notebook recipes/Getting_Started/Getting_Started_with_WatsonX.ipynb`

Now to put it in a pre-commit hook...

Comment (fayvor) [reply recipes/Getting_Started/Getting_Started_with_WatsonX.ipynb]: Fixed.

Comment (fayvor) [reply recipes/Getting_Started/Getting_Started_with_WatsonX.ipynb]: ```jupyter nbconvert --ClearOutputPreprocessor.enabled=True --inplace --to=notebook `find recipes|grep ".ipynb"` ```

**Comment by rawkintrevo** [comment]
lgtm

**Pull request merged**